lib/bob/job/build_elixir.ex

```diff
@@ -57,10 +36,7 @@ defmodule Bob.Job.BuildElixir do
       "v1.8" -> ["20.3", "21.3", "22.3"]
```

Comment:
I'm confused with this. Where is this ref coming from? I thought it was from `elixir-lang/elixir` tags, but there is not `v1.8` tag there, only full versions with `vMAJOR.MINOR.PATCH`

Reply:
It is the maintenance branch where we may backport fixes while the master branch will track the development for v1.9+.